src/constants.ts

```diff
@@ -3,29 +3,27 @@ export const NODE_ENV = process.env.NODE_ENV;
 export const AINFT_SERVER_ENDPOINT = {
   dev: 'https://ainft-api-dev.ainetwork.ai',
   prod: 'https://ainft-api.ainetwork.ai',
-}
+};
 
 export const AIN_BLOCKCHAIN_ENDPOINT = {
   dev: 'https://testnet-api.ainetwork.ai',
   prod: 'https://mainnet-api.ainetwork.ai',
-}
+};
 
-
-export const AIN_BLOCKCHAIN_CHAINID = {
+export const AIN_BLOCKCHAIN_CHAIN_ID = {
   dev: 0,
-  prod: 1
-}
+  prod: 1,
+} as const;
 
 export const MIN_GAS_PRICE = 500;
-export const APP_STAKING_LOCKUP_DURATION_MS = 30 * 1000 // 30 seconds
+export const APP_STAKING_LOCKUP_DURATION_MS = 30 * 1000; // 30 seconds
 
 export const SUPPORTED_AINFT_STANDARDS = {
   721: '721',
-}
+};
 
-export const SERVICE_NAME_MAP = new Map<string, string>([
-  ['openai', 'ainize_openai']
-]);
+export const THREAD_GC_MAX_SIBLINGS = 50;
```

Comment:
minor) if they do not look too small to you, how about using smaller values, e.g. (20, 10) pair? @jiyoung-an

Reply:
@platfowner i think 50 is too big too, so how about (30, 10)?

Reply:
30 also looks big to me. :) If possible, keeping the value as small as possible would help the budget management. And it looks like more than 20 threads cannot be rendered at once on the left-side panel?

Reply:
it's good!  I will be adjusted to (20, 10) in next pr! 
it seems appropriate to fetch 20 threads per call (similar rendered values)

Reply:
sounds great, thanks!